sources/install/package_web.sh

```diff
@@ -894,6 +894,19 @@ function install_postman() {
     add-history postman
     add-test-command "which postman"
     add-to-list "postman,https://www.postman.com/,API platform for testing APIs"
+
+function install_zap() {
+    colorecho "Installing ZAP"
+    zap_version=$(curl -s "https://www.zaproxy.org/download/" | grep -E -o "ZAP [0-9]+(\.[0-9]+)+" | head -1 | cut -d' ' -f2)
+    wget "https://github.com/zaproxy/zaproxy/releases/download/v${zap_version}/ZAP_${zap_version}_Linux.tar.gz" -O /tmp/zap.tar.gz
+    tar -xf /tmp/zap.tar.gz -C /tmp/
+    mv /tmp/ZAP_${zap_version} /opt/tools/ZAP/
```

Comment:
`mv /tmp/ZAP_${zap_version}/* /opt/tools/ZAP/`

Reply:
I'm not sure to understand. `mv /p1/d1 /p2/d2` will move d1 to /p2, doesn't it ?

Reply:
I would like you to move the **content** of the folder into /opt/ZAP and not the folder, because that would make /opt/ZAP/Zap_1.1.1/…

Reply:
Because /opt/ZAP is not a directory, it would actually rename Zap_1.1.1 in ZAP and move it into /opt/tools . But I did it your way if you prefer